### ISSUE TITLE
Incorrect foreground color for status badge-warning

### ISSUE THREAD

**Issue by karan-kang** [vmware/clarity#5259]
## Describe the bug

The foreground color of status badge warning does not have enough contrast with background.

## How to reproduce

1.  Visit: https://clarity.design/documentation/badges and observe the Status badges
2. ![image](https://user-images.githubusercontent.com/1234394/97998890-a8e99380-1db8-11eb-9cfd-8c6307bd87b9.png)

## Versions

Clarity: v4.0.5

**Comment by bdryanovski** [comment]
@karan-kang thank you for reporting this. We gonna try to add this issue in our next A11y bug fixes round.

**Comment by erabl-identos** [comment]
I've noticed some more changes on the badge coloring from v3 to v4.
The [release changelog](https://clarity.design/news/4.0.5) does not list any badge-related changes, so I'm wondering if the new colors are intended or if this is a side effect from another change?

<table>
<tr>
  <td>
    V3 <a href="https://v3.clarity.design/badges" target="_blank">https://v3.clarity.design/badges</a>
  </td>
  <td>
    V4 <a href="https://clarity.design/documentation/badges" target="_blank">https://clarity.design/documentation/badges</a>
  </td>
</tr>
<tr>
  <td>
    .badge-orange: #efc006;<br>
    .badge-light-blue: #79c6e6;<br>
    .badge-4: #efc006;<br>
    .badge-5: #79c6e6;<br>
    .badge-info: #0072a3;<br>
    .badge-success: #3c8500;<br>
    .badge-warning: #fee272;<br>
    .badge-danger: #c21d00;<br>
  </td>
  <td>
    .badge-orange: #613200;<br>
    .badge-light-blue: #179bd3;<br>
    .badge-4: #613200;<br>
    .badge-5: #179bd3;<br>
    .badge-info: #00567a;<br>
    .badge-success: #306b00;<br>
    .badge-warning: #613200;<br>
    .badge-danger: #991700;<br>
  </td>
</tr>
<tr>
  <td>
    <br><img src="https://user-images.githubusercontent.com/17832299/98547790-fa9e8c00-2298-11eb-9622-3e3879244007.png" width="300px"/>
  </td>
  <td>
    <br><img src="https://user-images.githubusercontent.com/17832299/98548192-7bf61e80-2299-11eb-84bf-df62503a1064.png" width="300px"/>
  </td>
</tr>
</table>

Also notice that `.badge-orange` and `.badge-light-blue` now have white text instead of black.

**Comment by mathisscott** [comment]
@erabl-identos 
These changes were intentional. They are to improve the accessibility (color contrast) of the badges.

**Comment by erabl-identos** [comment]
Thanks for the clarification.

**Comment by Jinnie** [comment]
@colinreedmiller can you please review the shown badge colors (warning especially) and give us the expected background/font color definitions.
Thank you.

**Comment by mme-private** [comment]
> These changes were intentional. They are to improve the accessibility (color contrast) of the badges.

Warning badge is almost unreadable, how does it improve something? And brown color for warning is also strange.

**Comment by Jinnie** [comment]
> > These changes were intentional. They are to improve the accessibility (color contrast) of the badges.
> 
> Warning badge is almost unreadable, how does it improve something? And brown color for warning is also strange.

Scott's comment was about the other minor differences, mentioned in the other comments above. The warning badge is obviously wrong. We're waiting for feedback from UX with the correct colors.

**Comment by jerone** [comment]
Any news from the UX team? Orange is not orange anymore, more like brown.

**Comment by Jinnie** [comment]
Feedback after my last talks with UX:
Due to accessibility color contrast requirements, this is the background color accepted as warning in Clarity Core:
See here: https://clarity.design/storybook/core/?path=/story/components-badge-stories--status
Synchronizing with them lead to this issue, as we still need to update our font color. We shouldn't mix light over dark with dark over light in the same component, as we initially did.

**Comment by gnomeontherun** [comment]
The fix here is to make the text white. The color contrast of text on orange is pretty hard to work out for accessibility requirements, hence the brownish color. This is only Clarity UI.
